third_party/docfx-doclet-143274/src/test/resources/expected-generated-files/com.microsoft.samples.Link.yml

```diff
@@ -114,10 +114,13 @@ references:
   fullName: "com.microsoft.samples.Link.Link"
   package: "com.microsoft.samples"
 - uid: "java.lang.String"
+  href: "https://docs.oracle.com/javase/8/docs/api/java/lang/String.html"
   spec.java:
   - uid: "java.lang.String"
     name: "String"
     fullName: "java.lang.String"
+    isExternal: false
```

Comment:
It seems like this is external?

Reply:
Same as above comment. As long as href is included, this is expected behavior.